CMakeLists.txt

```diff
@@ -335,6 +338,20 @@ if(GDBSTUB)
 	add_definitions(-DUSE_GDBSTUB)
 endif(GDBSTUB)
 
+if(VTUNE)
+	if(EXISTS "$ENV{VTUNE_AMPLIFIER_XE_2015_DIR}")
+		set(VTUNE_DIR "$ENV{VTUNE_AMPLIFIER_XE_2015_DIR}")
+	elseif(EXISTS "$ENV{VTUNE_AMPLIFIER_XE_2013_DIR}")
+		set(VTUNE_DIR "$ENV{VTUNE_AMPLIFIER_XE_2013_DIR}")
```

Comment:
Wasn't there a symlink or the like that had a consistent name (and linked to whatever the directory used by the current version was?